src/components/comment/ActionDropdown.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import Media from '../media/Media';
+import { MenuItem } from '../menu';
+
+import './ActionDropdown.scss';
+
+export interface ActionDropdownItem {
+    icon?: React.ReactNode;
+    onClick: () => void;
+    text: string;
+}
+
+export interface ActionDropdownProps {
+    items: ActionDropdownItem[];
+}
+
+function ActionDropdown({ items }: ActionDropdownProps) {
+    if (!items.length) {
+        return null;
+    }
+
+    return (
+        <Media.Menu data-testid="ThreadedComment-actionDropdown" className="bdl-ThreadedComment-actionDropdown">
```

Comment:
SUIT naming is component-specific, so this would be `bdl-ActionDropdown`.

Reply:
From the Comment's point of view the `ActionDropdown` is just a part of it. It's not a separate component and will not be used outside of `Comment`.